Tests/Fixtures/FooBundle/Controller/InvokableClassLevelController.php

```diff
@@ -11,11 +11,10 @@
 
 namespace Tests\Fixtures\FooBundle\Controller;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @Route(service="test.invokable_class_level.predefined")
```

Comment:
wasn't this meant to test the routing loader of this bundle ? Testing the Symfony one does not make sense here.

Reply:
I would assume no, as this test is part of the TemplateAnnotationTest.

Reply:
I think I originally created these tests to prevent regressions for the `@Template` annotation. I simply registered them as service this way, so my best guess is that this can be removed.